src/main/java/ml/shifu/shifu/core/dtrain/AbstractAdaptiveLearningRate.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * 
+ */
+package ml.shifu.shifu.core.dtrain;
```

Comment:
please use https://github.com/ShifuML/shifu/blob/master/shifu-codetemplates.xml and https://github.com/ShifuML/shifu/blob/master/shifu-formatter.xml

here is for license.